framework/Dockerfile

```diff
@@ -34,6 +33,30 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
+RUN git clone https://github.com/idrinth-api-bench/framework.git \
```

Comment:
you are still cloning into a random directory. Nothing below will work

Reply:
I thought the url came from the framework repo correct?

Reply:
the source was never the issue, the target is

Reply:
Are you referring to the mkdir?

Reply:
no, the cloning. You are cloning into whatever workdir was previously set